src/variants.ini

```diff
@@ -86,7 +86,7 @@
 
 ### Custom pieces
 # Custom pieces can be defined by using one of the available slots:
-# customPiece1, customPiece2, ..., customPiece26
+# customPiece1, customPiece2, ..., customPiece25
 # E.g., pawns without double steps could be described as:
 # customPiece1 = p:mfWcfF
```

Comment:
Somewhere around here we should document:

```
# You can define a custom king in the same way you can define another custom piece:
# E.g., to make a king that moves like a centaur:
# king = k:KN
```

You also created the `centaurking` example which is good :+1: But the top of this document is the closest I could find to a "language specification" for the variants.ini, so additionally putting a note here would be useful for documenting that.